dbt_subprojects/tokens/models/prices/prices_trusted_tokens.sql

```diff
@@ -18,6 +18,10 @@ WITH trusted_tokens AS (
                 ('abstract', 0x3439153EB7AF838Ad19d56E1571FBD09333C2809)
                 , ('abstract', 0x84a71ccd554cc1b02749b35d22f684cc8ec987e1)
                 , ('abstract', 0x000000000000000000000000000000000000800a)
+                , ('apechain', 0x48b62137edfa95a428d35c09e44256a739f6b557)  -- WAPE
+                , ('apechain', 0xe31c676d8235437597581b44c1c4f8a30e90b38a)  -- GNS
+                , ('apechain', 0xfc7b0badb1404412a747bc9bb6232e25098be303)  -- APE
+                , ('apechain', 0xcf800f4948d16f23333508191b1b1591daf70438)  -- ApeETH
```

Comment:
```suggestion
```

Reply:
lets play this safe and only do `WAPE` for now. when building DEXs, we can look to see what is swapped most and can be added here